packages/ckeditor5-mention/docs/_snippets/examples/chat-with-mentions.js

```diff
@@ -73,6 +73,19 @@ ClassicEditor
 		// Clone the first message in the chat when "Send" is clicked, fill it with new data
 		// and append to the chat list.
 		document.querySelector( '.chat-send' ).addEventListener( 'click', () => {
+			const message = editor.getData();
+
+			if ( !message ) {
+				const editorContainer = editor.sourceElement;
+
+				editor.editing.view.focus();
+
+				editorContainer.classList.add( 'highlighted' );
```

Comment:
You should do something like this instead:

```javascript
const editingView = editor.editing.view;

editingView.change( writer => {
	writer.addClass( 'highlighted', editingView.document.getRoot() );
} );
```

Once the editor starts, you should not use low-level DOM methods to modify its editing root (e.g. add or remove attributes) because they are under the control of the editor engine. Check out the `DowncastWriter` API for more methods (e.g. to remove the class) and docs.

Reply:
Fixed